TabulateSmarterTestPackage/Properties/PublishProfiles/Framework.pubxml

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
These profiles probably aren't needed but i was playing with various ways to deploy and run.